docs/UserGuide.md

```diff
@@ -417,7 +654,7 @@ Action | Format, Examples
   </tr>
   <tr>
     <td class="tg-0pky command"><code>findMember</code> / <code>findEvent</code></td>
-    <td class="tg-0pky">KEYWORD</td>
+    <td class="tg-0pky">MEMBER_NAME / EVENT_NAME / MORE_NAMES</td>
```

Comment:
Should this stay as the same for the 2103T side since our error message says KEYWORD
```suggestion
    <td class="tg-0pky">KEYWORD / MORE_KEYWORDS</td>
```

Reply:
I will copy over the table from 2101 side after its done